### PR TITLE
bootloaderInstructions: add videos

### PR DESCRIPTION
**Pull request by dlech**
This adds videos to show how to enter bootloader mode.

Issue: https://github.com/pybricks/support/issues/728

### CODE REVIEW

File: src/firmware/bootloaderInstructions/assets/bootloader-essentialhub-metadata.vtt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+00:00.100 --> 00:05.000
+
+wait-for-light
+00:05.000 --> 00:09.000
+
+connect-usb
+00:09.000 --> 00:15.000
+
+release-button
+00:15.000 --> 00:27.000
```
Comment (dlech) [inline src/firmware/bootloaderInstructions/assets/bootloader-essentialhub-metadata.vtt:13]: This is quite long for a short step, we could cut at least 5 seconds off the end of the video.

File: src/firmware/bootloaderInstructions/assets/bootloader-primehub-metadata.vtt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+00:00.100 --> 00:07.000
+
+connect-usb
+00:07.000 --> 00:13.000
+
+wait-for-light
+00:13.000 --> 00:19.000
+
+release-button
+00:19.000 --> 00:29.000
```
Comment (dlech) [inline src/firmware/bootloaderInstructions/assets/bootloader-primehub-metadata.vtt:13]: This one is a bit long too and we could cut 5 seconds off the video.

**Comment by laurensvalk** [comment]
I could try looking at the details later tonight, but if you’re happy with how it looks I’m all 🟢 for a beta/RC release.

We can tweak the videos and text next week as needed.

File: src/firmware/bootloaderInstructions/assets/bootloader-essentialhub-metadata.vtt (hunk 1 above)
Comment (dlech) [reply src/firmware/bootloaderInstructions/assets/bootloader-essentialhub-metadata.vtt]: ffmpeg to the rescue

**Pull request merged**